### ISSUE TITLE
[Table UI filters] multiple dropdown filters same values

### ISSUE THREAD

**Issue by stijntilleman** [pyrocms/pyrocms#4036]
I'm trying to implement a couple of dynamic filters on an overview page. By their self the filters work perfectly, but when I try to combine them on a page, both filters use the values defined for the first filter in the filter definition.

```php
 # in CustomTableBuilder
 public function setFilters($filters)
    {
        $filters = [
            "company" => [
                "slug" => "company",
                "filter" => "select",
                "placeholder" => trans("module::field.company.name"),
                "query" => CompanyFilter::class,
                "options" => dispatch(new CompanyOptionsHandler),
            ],
            "supplier_status" => [
                "slug" => "status",
                "filter" => "select",
                "placeholder" => trans("module::field.supplier_status.name"),
                "query" => SupplierStatusFilter::class,
                "options" => dispatch(new SupplierStatusOptionsHandler),
            ],
        ];

        return parent::setFilters($filters);
    }
```

The same is true when I add this to the `CustomTableBuilder.filters` property

```php
# in CustomTableBuilder
protected $filters = [
        "company" => [
            "slug" => "company",
            "filter" => "select",
            "placeholder" => "Company",
            "query" => CompanyFilter::class,
            "options" => [
                "hq" => "Headquarters",
                "sub" => "Sub",
            ],
        ],
        "supplier_status" => [
            "slug" => "status",
            "filter" => "select",
            "placeholder" => "Status",
            "query" => SupplierStatusOptionsHandler::class,
            "options" => [
                "open" => "Open",
                "closed" => "Closed",
            ],
        ],
    ];
```

In the last case both selects get the company values.

**Comment by RyanThompson** [comment]
Can you dump the built objects from the `FilterBuilder`? Also what's the post data they are sending?

**Comment by aliselcuk** [comment]
Looks like this can be merged with #4087

**Comment by RyanThompson** [comment]
fixed: https://github.com/anomalylabs/streams-platform/commit/30611b459d7e6ac0575eee1a8fea833ab5d58ab3
